### PR TITLE
[IMP] Python3 support

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

**Comment by rousseldenis** [comment]
This seems to build.

@legalsylvain @PierrickBrun I based this on #29

**Comment by PierrickBrun** [comment]
Hi @rousseldenis , it looks like it only works with python3 now. Would it be possible to have it working for both python 2 and 3 ?

**Comment by rousseldenis** [comment]
> Hi @rousseldenis , it looks like it only works with python3 now. Would it be possible to have it working for both python 2 and 3 ?

Ok, I'll make corrections

**Comment by rousseldenis** [comment]
@PierrickBrun After thinking, python3 lib should not be retro-compatible.

We should eventually make a release with some missing PR's and then release a new one with python3. What do you think ?

**Comment by PierrickBrun** [comment]
Hi @rousseldenis . I've forgot to answer about this, but I agree, we should drop python 2 support. I think I will make a new "Python 2" branch and use the master for python3. Are you ok with the current state of pywebdriver or do you want some PRs to be merged before switching to python3 ?

**Comment by rousseldenis** [comment]
> Hi @rousseldenis . I've forgot to answer about this, but I agree, we should drop python 2 support. I think I will make a new "Python 2" branch and use the master for python3. Are you ok with the current state of pywebdriver or do you want some PRs to be merged before switching to python3 ?

Hi,

I think, the faster we switch to Python3, the better we will be, so go on ! :smiley: 

All PR's I've opened are done to be used with p3.

File: pywebdriver/views.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def status():
 
 @app.route('/usb_devices.html', methods=['GET'])
 def usb_devices():
-    str_devices = commands.getoutput("lsusb").split('\n')
+    str_devices = subprocess.getoutput("lsusb").split('\n')
```
Comment (sbidoul) [inline pywebdriver/views.py]:
```suggestion
    str_devices = subprocess.check_output("lsusb", universal_newlines=True).split('\n')
```

Comment (bealdav) [reply pywebdriver/views.py]: @rousseldenis any idea about this suggestion ? Thanks

Comment (rousseldenis) [reply pywebdriver/views.py]: @bealdav I think the suggestion is good.
@sbidoul As i read in docs https://docs.python.org/fr/3.7/library/subprocess.html
wouldn't be useful to catch CalledProcessError ?

**Comment by PierrickBrun** [comment]
Hi @rousseldenis I continued the work on #49 , I think we can close this one as it targets `gunicorn-bionic` which have been merged

**Pull request closed without merging**